src/peft/utils/other.py

```diff
@@ -313,6 +316,28 @@ def _get_batch_size(input_ids: Optional[torch.Tensor], inputs_embeds: Optional[t
     return batch_size
 
 
+def default_forward(self, *args, **kwargs):
+    return self.get_base_model()(*args, **kwargs)
+
+
+def update_forward_signature(model) -> Union[torch.nn.Module, PushToHubMixin]:
```

Comment:
Regarding the signature, we should be able to say that `model` is always a `PeftModel`, right? Also, I don't think we need any return value, since we're modifying the model inplace.

On another note, could this be generalized to work with `generate` too? Note that `generate` doesn't have `*args`.

Reply:
In order to have the `def update_forward_signature(model:PeftModel):` this would have to be put in a different file (helpers) outside utils, otherwise we run in to a circular import.
I'm sure something could be done for the generate mehod too, the main issue with it I see right now would be that the base PeftModel doesn't have a generate method, we would probably have to use the one from PeftModelForCausalLM